bench/bench.ml

```diff
@@ -60,7 +60,28 @@ let main () =
         ignore (Yojson.Safe.seq_to_string ~buf stream)
       )
     end;
-  ])
+  ]
+
+let buffer =
+  let buf = Buffer.create 4096 in
+  let data = large_int_assoc in
+  Bench.make_command [
+    Bench.Test.create ~name:"JSON writing with internal buffer" (fun () ->
+      Out_channel.with_file "/dev/null" ~f:(fun oc ->
+        ignore (Yojson.Safe.to_channel oc data)));
+    Bench.Test.create ~name:"JSON writing with provided buffer" (fun () ->
+      Out_channel.with_file "/dev/null" ~f:(fun oc ->
+        (* make extra sure the buffer is empty *)
+        Buffer.clear buf;
```

Comment:
Maybe, when #132 is merged, these two lines can be removed?
As it's a benchmark, we do not want to do extra steps than what is necessary.
I think we should wait for #132 to be merged, and rebase this PR on top of it, without these lines.

Reply:
I merged #132 and rebased upon it, thus removing the `Buffer.clear`.